### DISCUSSION TITLE
Docker image  suspend in startup.

### DISCUSSION THREAD

**Discussion by caffeine-cn** [bfenetworks/bfe#1182] [General]
**Describe the bug**
follow the install instruction https://www.bfe-networks.net/zh_cn/installation/install_using_docker/
docker run suspend in  sample code  
`docker run -p 8080:8080 -p 8443:8443 -p 8421:8421 bfenetworks/bfe` with information
`maxprocs: Leaving GOMAXPROCS=56: CPU quota undefined`

**To Reproduce**
install with  docker   with sample code `docker run -p 8080:8080 -p 8443:8443 -p 8421:8421 bfenetworks/bfe`

**Expected behavior**
docker run successful.

**Comment by zhugelianglongming** [comment]
It seems like Golang program in your docker environment runs with too much logical processors.

You can try to set `GOMAXPROCS` in docker env to be likes cores for container(cpu quota).
`run -p 8080:8080 -p 8443:8443 -p 8421:8421 -e GOMAXPROCS="${expected_value}" bfenetworks/bfe` 

> More:
> [关于GOMAXPROCS的设置](https://blog.csdn.net/shzcy/article/details/123892218)

**Comment by liangyuanpeng** [comment]
--
